lib/zxcvbn/tester.rb

```diff
@@ -21,15 +23,21 @@ def initialize
     end
 
     def test(password, user_inputs = [])
-      PasswordStrength.new(@data).test(password, user_inputs)
+      PasswordStrength.new(@data).test(password, sanitize(user_inputs))
```

Comment:
Adding some specs below revealed that non-string values can be added to the dictionary and they cause a similar runtime error to the one that caused me to open this gem.